src/contributorFetcher.ts

```diff
@@ -0,0 +1,26 @@
+import { GitHubContributorFetcher } from './gitHubContributorFetcher';
+import { GithubSponsorable, SponsorableFetcher, queryTemplate } from './sponsorableFetcher';
+
+export class ContributorFetcher {
+    public async fetchContributors(
```

Comment:
Since we have a class here, I generally find it better to put the configuration stuff into a constructor, separate from the call that actually does the work. In this case, my first guess would be to pass the token to the constructor, but to pass the repo identifiers into `fetchContributors` (aka `findContributors`).